styles/modules/modals/_wallet.scss

```diff
@@ -224,8 +215,23 @@
 }
 
 .spendConfirmBox {
-  .spinner {
+  .sendConfirm {
     position: relative;
-    top: -10px;
+
+    .spinner {
+      position: relative;
+      top: -10px;
+    }
+  }
+
+  &.centeredBelow {
+    // The container that you want the spendConfirmBox to be
+    // centered below must be non-statically positioned.
+    .sendConfirm {
+      margin-top: 10px;
```

Comment:
I don't think you want the 10px margin here. On my Windows computer it pushes the tooltip down too far:

![image](https://user-images.githubusercontent.com/1584275/36440786-8f061e86-163e-11e8-969b-c6e94ea5e5f8.png)

Without the margin it looks like this:
![image](https://user-images.githubusercontent.com/1584275/36440810-9dec86f6-163e-11e8-80b2-0fec8c8a3fd2.png)

Reply:
The margin is needed in the order details though.

It looks like purchase.scss changes the top value to calc(100% + 10px), you could either take that out and use the 10px margin instead, or apply the same top value in the purchase and in the order details.